ci/cron/src/Main.hs

```diff
@@ -325,43 +221,62 @@ fetch_gh_paginated url = do
                 (_, _, _, [url, rel]) -> (rel, url)
                 _ -> fail $ "Assumption violated: link header entry did not match regex.\nEntry: " <> l
 
-fetch_gh_versions :: IO ([GitHubVersion], GitHubVersion)
+data Versions = Versions { top :: String, all_versions :: Set.Set String, dropdown :: [String] }
+    deriving Eq
+
+versions :: [Version] -> Versions
+versions vs =
+    let to_strings = map (Data.SemVer.toString . tag)
+        all_versions = Set.fromList $ to_strings vs
+        dropdown = to_strings $ List.sortOn (Data.Ord.Down . tag) $ filter (\v -> tag v >= parse (Text.pack "1.0.0") && not (prerelease v)) vs
+        top = head dropdown
+    in Versions {..}
+
+fetch_gh_versions :: IO Versions
 fetch_gh_versions = do
-    resp <- fetch_gh_paginated "https://api.github.com/repos/digital-asset/daml/releases"
-    let latest = List.maximumOn (to_v . name) $ filter (not . prerelease) resp
-    return (resp, latest)
+    response <- fetch_gh_paginated "https://api.github.com/repos/digital-asset/daml/releases"
+    -- versions prior to 0.13.10 cannot be built anymore and are not present in
+    -- the repo.
+    return $ versions $ filter (\v -> tag v >= parse (Text.pack "0.13.10")) response
```

Comment:
Do we still need this filter?

Reply:
I have a slight preference for keeping it. Keeping it means the combination of `versions.json` and `hidden.json` list all the versions for which we have docs on the docs site. Not keeping it means that same combination instead represents all the versions for which we are not going to try building the docs again. It's a small distinction but I like the former better.

If we did want to remove it (I could be persuaded, but not as part of this PR), we'd need some sort of plan with an intermediate state in this script to write the new, more complete `hidden.json` while not trying to build them.

Reply:
Makes sense, let’s keep it then.